code/game/machinery/doppler_array.dm

```diff
@@ -19,32 +19,39 @@ var/list/doppler_arrays = list()
 	return ..()
 
 /obj/machinery/doppler_array/proc/sense_explosion(var/x0,var/y0,var/z0,var/devastation_range,var/heavy_impact_range,var/light_impact_range)
-	if(stat & NOPOWER)	return
-	if(z != z0)			return
+	if(stat & NOPOWER)
+		return
+	if(z != z0)
+		return
 
 	var/dx = abs(x0-x)
 	var/dy = abs(y0-y)
 	var/distance
-	var/direct
 
 	if(dx > dy)
 		distance = dx
-		if(x0 > x)	direct = EAST
-		else		direct = WEST
 	else
 		distance = dy
-		if(y0 > y)	direct = NORTH
-		else		direct = SOUTH
 
 	if(distance > 100)
 		return
 
-	if(!(direct & dir))
-		return
-
 	var/message = "Explosive disturbance detected - Epicenter at: grid ([x0],[y0],[z0]). Epicenter radius: [devastation_range]. Outer radius: [heavy_impact_range]. Shockwave radius: [light_impact_range]."
 	global_announcer.autosay(message, src.name)
 
+	var/list/gained_tech
+	switch(devastation_range)
+		if(-INFINITY to 1) // minimum
+			gained_tech = list(TECH_BLUESPACE = 1, TECH_COMBAT = 2, TECH_MATERIAL = 2)
+		if(1 to 2) // starter bombs
+			gained_tech = list(TECH_BLUESPACE = 2, TECH_PHORON = 2, TECH_COMBAT = 3, TECH_MATERIAL = 3)
```

Comment:
That might be a bit too much tech-wise as that would also give these tech levels, just for dropping drills.

Reply:
just tested it, drill droppers only cause one devastation, which provides
![image](https://user-images.githubusercontent.com/22774890/109799634-ee15b900-7c24-11eb-98a3-335f4e1514cd.png)